man/bundle-init.ronn

```diff
@@ -12,7 +12,13 @@ adding a [`Gemfile(5)`][Gemfile(5)] to a gem with a gemspec, the `--gemspec` opt
 automatically add each dependency listed in the gemspec file to the newly
 created [`Gemfile(5)`][Gemfile(5)].
 
+One of the lines included in the default [`Gemfile(5)`][Gemfile(5)] generated is `# frozen_string_literal: true`. This is a magic comment supported for the first time in Ruby 2.3. The presence of this line results in all string literals in the file being implicitly frozen. Frozen string literals will be the default in Ruby 3.
```

Comment:
You need to split this line up to match the max width, which is about 80 characters. Checkout the other man pages to see what i'm talking about.

Reply:
Thanks, have corrected